sdk/communication/AzureCommunicationChat/Package.swift

```diff
@@ -24,7 +24,7 @@ let package = Package(
     dependencies: [
         .package(
             url: "https://github.com/Azure/azure-sdk-for-ios.git",
-            .branch("master")
+            .branch("dev/AzureCore")
```

Comment:
Was this change suppose to point to this branch?

Reply:
This was generated so I manually reverted it for now, referenced on autorest.swift here: https://github.com/Azure/autorest.swift/issues/310